src/components/Kv/KvMap.vue

```diff
@@ -0,0 +1,297 @@
+<template>
+	<div class="tw-relative tw-block tw-w-full" :style="mapDimensions">
```

Comment:
Since you have the aspect ratio provided as a prop, you could use the padding-bottom trick to automatically set the height `padding-bottom: ${100 / aspectRatio}%;`. You would also need to add the `tw-absolute` class to the map element so that it would cover up the padding. That way this component will always take up the right amount of space on the page, even before it loads.

Reply:
Good idea, I'll give this a try!

Reply:
Ok I've added that in the computed props along with override handling for specified height and width values in via props.